nemo/collections/asr/modules/conformer_modules.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import torch.nn as nn
+
+from nemo.collections.asr.modules.activations import Swish
+
+__all__ = ['ConformerConvolution', 'ConformerFeedForward']
+
+
+class ConformerConvolution(nn.Module):
```

Comment:
I would urge you to subclass NeuralModule and add type checking. Alternatively, please move this to parts as opposed to keeping it inside the modules directory.

Reply:
Moved.